netsim/modules/lag.yml

```diff
@@ -34,14 +34,16 @@ attributes:
         min_value: 1
         _alt_types: [ bool ]
       mac: mac
-
-  # interface:     # Internal, don't declare as official attribute
-    # _mlag: bool
+  interface:
+    lacp_system_id:
```

Comment:
I think this is a typical candidate for auto-generation; users will probably care even less about this value, than the IP addresses being used

It's good to allow it to be specified, but I think the typical case should be auto-generated (e.g. based on node ID, or peergroup number).

Similarly, for manual specification we would probably want to allow node level - and then copy it to all the lag interfaces. It would typically be the same consistent value everywhere, no?